File: 23WH1A12C1/Grades..c
#include<stdio.h>

int main()
{
    int phy,math,pps;
    float per;
    printf("Enter 3 sub marks: ");
    scanf("%d %d %d", &phy,&math,&pps);
    per=(phy+math+pps)/3.0;
    printf("percentage=%2f \n",per);
    if(per>=70)
     printf("Distinction");
    else if(per=60 || per<70)
     printf("First class");
    else if(per=40 || per<60)
     printf("Second class");
    else
     printf("Failed");    
    return 0;
}

File: 23HW1A1268/Python/Grades.py
marks = int(input("Enter your marks: "))
if (90 < marks) & (marks <= 100):
    print("Grade O")
elif (75 < marks) & (marks <= 90):
    print("Grade A")
elif (60 < marks) & (marks <= 75):
    print("Grade B")
elif (45 < marks) & (marks <= 60):
    print("Grade C")
else:
    print("Grade D")